spec/requests/vendor_api/1.1/get_multiple_applications_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
```

Comment:
hmm I wonder if the presenter spec is enough to test this, and when we create/update/cancel interviews we can add the request specs?

Reply:
I don't mind removing this but see https://github.com/DFE-Digital/apply-for-teacher-training/pull/6387#discussion_r790783469 for what this is trying to do.

Reply:
I think we could possibly move this test (the one testing the empty array to the application presenter) and it could be covered there as well? This way we remove the single and multiple request spec and reserve those solely for the actions we intend to add. I think after we revisit the v1 tests in the end we can bring them inline with those tests, and when we validate the schema as well it will cover the different scenarios

Reply:
Removing the multiple applications spec for now, we can revisit at the end of this epic. Have not removed the single application request spec, because this is the only way we currently test the concern is included in `v1.1`.